test/jdk/java/awt/Focus/ModalDialogInFocusEventTest.java

```diff
@@ -22,11 +22,11 @@
  */
 
 /*
-  test
+  @test
+  @key headful
   @bug 4531693 4636269 4681908 4688142 4691646 4721470
   @summary Showing modal dialog during dispatching SequencedEvent
-  @key headful
-  @run main AutomaticAppletTest
+  @run main ModalDialogInFocusEventTest
```

Comment:
This test can be run standalone (`java ...`), so I prefer to keep it that way, so I didn't add the usage of `Platform.isOnWayland`.